Pod/Classes/VOKCoreDataManager.m

```diff
@@ -90,7 +92,11 @@ - (NSManagedObjectContext *)managedObjectContext
     if (!_managedObjectContext) {
         NSAssert([NSOperationQueue currentQueue] == [NSOperationQueue mainQueue],
                  @"Must be on the main queue when initializing main context");
-        _managedObjectContext = [self managedObjectContextWithConcurrencyType:NSMainQueueConcurrencyType];
+        self.privateRootContext = [self managedObjectContextWithConcurrencyType:NSPrivateQueueConcurrencyType
+                                                           parentContext:nil];
```

Comment:
Also: should this be moved into it's own lazy-loading accessor method? I could go either way on that one.